deploy/helm/kubernetes/templates/scheduller-deployment.yaml

```diff
@@ -22,24 +22,22 @@ spec:
         - kube-scheduler
         - --authentication-kubeconfig=/etc/kubernetes/scheduler.conf
         - --authorization-kubeconfig=/etc/kubernetes/scheduler.conf
-        - --bind-address=127.0.0.1
+        - --bind-address=0.0.0.0
         - --kubeconfig=/etc/kubernetes/scheduler.conf
         - --leader-elect=true
-        image: k8s.gcr.io/kube-scheduler:v1.18.0
-        imagePullPolicy: IfNotPresent
+        image: {{ $.Values.scheduller.image }}
```

Comment:
Is there any specific reason to call the root `$`?

Reply:
fix